client/my-sites/post-type-list/post-actions-ellipsis-menu/qrcode.jsx

```diff
@@ -0,0 +1,73 @@
+import { Icon } from '@wordpress/components';
+import { Component } from '@wordpress/element';
+import { localize } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import PopoverMenuItem from 'calypso/components/popover-menu/item';
+import { getPost } from 'calypso/state/posts/selectors';
+import PostActionsQRCode from '../../post-actions-qr-code';
+
+class PostActionsEllipsisMenuQRCode extends Component {
+	static propTypes = {
+		globalId: PropTypes.string,
+		translate: PropTypes.func.isRequired,
+		status: PropTypes.string,
+	};
+
+	constructor( props ) {
```

Comment:
In case we decided to stay with a class component, the `constructor()` is unnecessary. We could inline the default state directly in the component class:

```
state = {
	showQRCode: false,
};
```